### ISSUE TITLE
[UX] Add a node/content block in the "Home page" layout?

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4867]
During the Dec. 31, 2020 UX/design meeting, there's an interesting discussion, which I would title "panels vs a node for the home page" (45min into the call): https://youtu.be/Fc-oQ55LQi0?t=2722

The main topic of this discussion was why use a layout for the home page, instead of simply using a node.

@jenlampton mentioned that the panel/layout solution was the most common use case. I'm not sure about that without having any hard telemetry data to support it, but as @stpaultim  mentioned in the call, I also find the Backdrop way much more powerful/flexible than what we had in D7.

I also believe that the fact that everything rendering-wise in Backdrop is handled by layouts is what dictated the way we implemented things. This might confuse people that come from D7 though, and as Mahmudul and @philsward who were in the call mentioned, they may prefer the simplistic way of D7 for the home page, where they could simply add a specific node, and then edit that node instead of having to learn how things work with the Home page layout (and layouts in general).

@jenlampton kept saying "Sure, you can do that too if you like" throughout this discussion, but what was missed was how hard that may be for people that are new to how layouts work in Backdrop.

Another thing that we keep forgetting is that since v1.5.0 we have introduced content blocks in Backdrop! (see #1076) ...this means that people can now add a node block to their Home page layout, pointing to any piece of content that they want to use as their "Home" node. They can then edit that piece of content, pretty much D7-style 👍 🎉 (and they can change that if they want, once they have become more familiar with how layouts/blocks work in Backdrop) ...the question is: do they know how to do that though?

So this issue is to discuss how to better introduce people to this whole thing:

- Should we change the Home page layout, to include such a node block OOTB?
- Should we create documentation in b.org, or a video tutorial that shows how to do that?
- Should we then link to that documentation/tutorial? ...and if so, then:
  - add it in an info message at the top of the Home page layout?
  - mention it in the help text in the site settings form?
  - provide it as a distinct option in the site settings form?
  - add it as content in the node that the node block points to?
- How does this whole thing tie into other issues like #4849 and/or #4839

Any other thoughts/ideas?

**Comment by klonos** [comment]
Another couple of related ideas:

- Back in 2016 I proposed #1936. If we had the ability to disable blocks, and given that it is easier for people to enable and tweak existing things instead of having to add/create/configure them from scratch, then perhaps we could add a disabled view block + a disabled node block in the Home page layout OOTB.

- Perhaps the various options provided in the "Home page" fieldset in `admin/config/system/site-information` (where you set home page path) could be converted into a "special" block (like the "Page title" block, which is then only made available for the Home page layout.

**Comment by indigoxela** [comment]
> Should we change the Home page layout, to include such a node block OOTB?

That's actually something that @olafgrabienski already suggested [in the other issue](https://github.com/backdrop/backdrop-issues/issues/4849#issuecomment-753541167) and he wrote that he's "planning to make a feature request for such a third core content type..." (node type with hidden path, exclusively for this use case). :wink: And I support that idea.

Using existing content in layouts is (probably) the quickest way to construct landing pages and it's still easy to handle, with a familiar workflow (node forms).

But most of all: less privileged users can edit these content snippets without the need (or permission) to also edit the layout. This is something every site builder should know. And I'm not sure if they do.

**Comment by klonos** [comment]
> But most of all: less privileged users can edit these content snippets without the need (or permission) to also edit the layout.

Excellent point @indigoxela 👍

**Comment by philsward** [comment]
The solution I would advocate for is having text on the home page that directs new builders to check out the home layout.

Whether this content sits in a block or in a node, doesn't really matter but just having something there that says:

"Want to change how your homepage looks? Go here!

Want to add your own text? Simply edit me and add your own!"

In my mind, pointing people in the right direction of how to manipulate the home page from instructions on the home page will go a long way.

**Comment by stpaultim** [comment]
> How does this whole thing tie into other issues like #4849 and/or #4839

#4839 is intended to be a quick fix to help users while bigger and more long term solutions are considered and implemented. 

In some ways this issue could be a duplicate of #4849, except that this issue is framed slightly different. The intent of the issue was MAINLY to do exactly what this one suggests.  

That issue (#4849) suggests, but is not insistent upon, replacing the promoted content block with a node bloc and this one doesn't. I think both options are worth considering. I am very interested in the idea of a new content type for "info" nodes that can be used for blocks. Using this new content type for the new home page node makes sense to me. I'd like to hear more discussion on this idea (a new content type).

**Comment by klonos** [comment]
> replacing the promoted content block with a node bloc and this one doesn't.

Yeah, I think that this is the main difference then.

> I am very interested in the idea of a new content type for "info" nodes that can be used for blocks.

Is the idea to set that content type to be "rabbit-holed", so that it cannot be viewed directly? It feels like something like that would basically create a poor-man's paragraphs-style thingy, where the Home page is comprised by one or more content blocks of this new type.

Having said that, I think that adding a new content type might complicate things.

**Comment by olafgrabienski** [comment]
> Is the idea to set that content type to be "rabbit-holed", so that it cannot be viewed directly?

Yes, see https://github.com/backdrop/backdrop-issues/issues/4849#issuecomment-753541167.

> ... I think that adding a new content type might complicate things.

I was planning to make the feature request for a third core content type independently of the homepage question, but can do it with the homepage in mind, as one possible use-case. We shouldn't wait for it, but if/when it looks suitable, we can always consider it later.

**Comment by stpaultim** [comment]
@klonos - what do you think about closing this issue. In my view, this was an alternative to https://github.com/backdrop/backdrop-issues/issues/4903 which was just merged, BUT this idea does address some slightly different needs and might still be worth considering. 

My preference would be to close this issue and start fresh if someone decides they want to pursue something like this.
